gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTable.java

```diff
@@ -86,6 +96,117 @@ public KnoxShellTable value(Comparable<? extends Object> value) {
     return col;
   }
 
+  public Conversions getConversion(Comparable<? extends Object> colIndex) {
+      Conversions type = null;
+
+      if (colIndex instanceof Double) {
+        type = Conversions.DOUBLE;
+      }
+      else if (colIndex instanceof Integer) {
+        type = Conversions.INTEGER;
+      }
+      else if (colIndex instanceof Float) {
+        type = Conversions.FLOAT;
+      }
+      else if (colIndex instanceof Byte) {
+        type = Conversions.BYTE;
+      }
+      else if (colIndex instanceof Short) {
+        type = Conversions.SHORT;
+      }
+      else if (colIndex instanceof Long) {
+        type = Conversions.LONG;
+      }
+      return type;
+    }
+
+    public double[] toDoubleArray(String colName) throws IllegalArgumentException {
```

Comment:
Can you add some javadocs here since its a public method?

Reply:
Changed to private (no need to be public) and added javadocs to the other public methods (mean, median, etc).